lib/auth0.js

```diff
@@ -3,12 +3,15 @@ import { initAuth0 } from '@auth0/nextjs-auth0';
 export default initAuth0({
   clientId: process.env.AUTH0_CLIENT_ID,
   clientSecret: process.env.AUTH0_CLIENT_SECRET,
+  audience: process.env.AUTH0_AUDIENCE,
   scope: process.env.AUTH0_SCOPE,
   domain: process.env.AUTH0_DOMAIN,
   redirectUri: process.env.REDIRECT_URI,
   postLogoutRedirectUri: process.env.POST_LOGOUT_REDIRECT_URI,
   session: {
     cookieSecret: process.env.SESSION_COOKIE_SECRET,
     cookieLifetime: process.env.SESSION_COOKIE_LIFETIME,
+    // cookieDomain: process.env.SESSION_COOKIE_DOMAIN,
```

Comment:
We need to keep this around?

Reply:
yea.. I need to figure out how to better test and set that just in production. but I want to see this go first so I can inspect those cookies from that domain.